src/shell/pty.c

```diff
@@ -62,6 +65,163 @@ shell_terminus_server_start (flux_shell_t *shell, const char *shell_service)
     return t;
 }
 
+static void pty_monitor (struct flux_pty *pty, void *data, int len)
+{
+    flux_plugin_arg_t *args = flux_plugin_arg_create ();
```

Comment:
check for NULL return from `flux_plugin_arg_create()`

Edit: oh, I guess `flux_plugin_arg_pack()` will fail with EINVAL in that case so maybe OK.